docs/features/api.md

```diff
@@ -0,0 +1,53 @@
+---
+title: API instructions
+---
+
+# intro
+
+> skip the intro if you are familiar with the datapackage concept.
+
+One of ideas of the datahub.io project is to store the data together with its description (the metadata). Data+metadata are called the *datapackage* or *dataset*. Detailed description of the *datapackage* format is here: http://datahub.io/docs/data-packages
+
+So usually a dataset consist of:
+- **Metadata**: descriptor file, named `datapackage.json`, it contains all the information that you need to find and to use the data itself
+- **Data**: data file(s). Usually it is tabular (csv, xls) but could be any file type.
+
+# GET API
+
+### 1. get files via `/r/` endpoint
```

Comment:
Let's use capitalize again

Reply:
Do we really need to enumerate it? My first instinct was that this is the first step and then comes the second one.

Reply:
Also, I think we need level 2 heading here instead of 3.